tests/test_cli.py

```diff
@@ -1,4 +1,5 @@
 from pathlib import Path
+from unittest import result
```

Comment:
This isn't being used. All instances of `result` below are variables returned by `runner.invoke`.

Reply:
I think this line was inserted by VSCode, which I'm not happy about. I need to figure out how to turn this "feature" off. The same thing happened with `isfile`. Maybe there's such a thing as too much automation...

Reply:
I removed this import line

Reply:
> I think this line was inserted by VSCode, which I'm not happy about. 

Ugh. That's annoying, all the more so because it keeps doing it ☹️ 

> Maybe there's such a thing as too much automation...

😝